src/data-types/numbers.js

```diff
@@ -6,22 +6,22 @@ const numThree = 32
 // TODO: Add code below using Javascript numeric operators so that the tests pass
 
 // Set this variable to numOne added to numTwo
-const numOnePlusNumTwo = NaN
+const numOnePlusNumTwo = 8 + 16
```

Comment:
can you re-write the solutions below to use the variables at the start of the file?

for example: `const numOnePlusNumTwo = numOne + numTwo`